app/src/main/java/com/codesoom/assignment/controllers/UserController.java

```diff
@@ -0,0 +1,64 @@
+package com.codesoom.assignment.controllers;
+
+import com.codesoom.assignment.application.UserService;
+import com.codesoom.assignment.domain.User;
+import com.codesoom.assignment.dto.UserData;
+import org.springframework.http.HttpStatus;
+import org.springframework.web.bind.annotation.CrossOrigin;
+import org.springframework.web.bind.annotation.DeleteMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.ResponseStatus;
+import org.springframework.web.bind.annotation.RestController;
+
+import javax.validation.Valid;
+
+@RestController
+@RequestMapping("/user")
+@CrossOrigin
+public class UserController {
+
+    private final UserService userService;
+
+    public UserController(UserService userService) {
+        this.userService = userService;
+    }
+
+    /**
+     * userData로 User를 생성하고, 생성된 User와 201(Created)를 응답합니다.
```

Comment:
이 정도로도 충분합니다. `CREATED`로 응답한다는 건 애노테이션에도 있는 정보에요. 애노테이션의 정보는 일종의 메타 정보이므로 JavaDoc에 굳이 작성해주지 않아도 됩니다. JavaDoc을 작성할 때에는 가급적 코 드 에 만 집중하시면 됩니다.
```suggestion
     * userData로 User를 생성하고, 생성된 User를 리턴합니다.
```

Reply:
네! 확인감사합니다.
javadoc 을 작성하면서 Service 와 Controller 가 똑같은 내용이라고 생각이 들었는데요.
거의 비슷하게 작성이 되는건가요???

Reply:
오 너무너무 좋은 질문이에요. 지금 서비스와 컨트롤러에서 JavaDoc을 작성해보니까 내용이 완전히 똑같죠? 그건 지금 컨트롤러와 서비스가 거의 1:1 관계로 코딩이 되어 있기 때문입니다. 만약 컨트롤러가 여러 개의 서비스를 사용하게 되거나, 하나의 서비스가 여러 컨트롤러에서 호출되는 구조가 되면 JavaDoc도 많이 달라질 겁니다. 

기왕 하는 김에 서비스 하나를 골라서 둘로 쪼개보는 연습을 해도 괜찮을 겁니다.

Reply:
아하 이해했습니다. 회사 소스에 대부분 javaDoc이 작성되어 있지 않은데 한번 연습해봐야겠네요 ㅎㅎ 감사합니다.